packages/conventional-changelog-writer/test/index.spec.js

```diff
@@ -305,6 +305,7 @@ describe('conventionalChangelogWriter', function () {
         }))
         .pipe(through(function (chunk, enc, cb) {
           expect(chunk.toString()).to.contain('# 1.0.0 ')
+          console.log(chunk.toString())
```

Comment:
Was this just for testing? Can it be removed?

Reply:
Yes good catch, I'll remove